docs/dev-env.md

```diff
@@ -33,10 +38,11 @@ it ensures consistent and reproducible environment for all developers.
 
 ### Install Nix
 
-If you don't have it set up already,
-follow the instructions at: https://nixos.org/download.html
+You have 2 options to install nix:
+* [The official installer](https://nixos.org/download.html)
+* The [Determinate Nix Installer](https://zero-to-nix.com/start/install) which is maintained by a 3rd party, but is a little more user-friendly.
```

Comment:
People have trouble installing nix. Give them a 2nd option which worked well for me.

Reply:
> ### Enable nix flakes

This is not needed for Determinate nix installer.

Reply:
Thanks. Updated that section to mention this.